Misc/NEWS.d/next/Core and Builtins/2022-12-28-15-02-53.gh-issue-87447.7-aekA.rst

```diff
@@ -0,0 +1,2 @@
+Fix :exc:`SyntaxError` on comprehension rebind checking with names that are
```

Comment:
Can you add a bit more detail on what the problem is so readers that are not familiar know what's being changed.

This also needs a entry in the "what's new" for 3.11 probably

Reply:
Done! 
1. I've added "whatsnew" for 3.12
2. News entry is also updated